cli/src/cli/index.ts

```diff
@@ -166,94 +166,21 @@ export const runCli = async () => {
     if (cliResults.flags.nextAuth) cliResults.packages.push("nextAuth");
   }
 
-  const pkgManager = getUserPkgManager();
-
   // Explained below why this is in a try/catch block
   try {
     // if --packages flag is set, we are running in CI mode and should not prompt the user
     // if --default flag is set, we should not prompt the user
     if (!cliResults.flags.default && !CIMode) {
       if (!cliProvidedName) {
-        const { appName } = await inquirer.prompt<Pick<CliResults, "appName">>({
-          name: "appName",
-          type: "input",
-          message: "What will your project be called?",
-          default: defaultOptions.appName,
-          validate: validateAppName,
-          transformer: (input: string) => {
-            return input.trim();
-          },
-        });
-        cliResults.appName = appName;
-      }
-
-      const { language } = await inquirer.prompt<{ language: string }>({
-        name: "language",
-        type: "list",
-        message: "Will you be using JavaScript or TypeScript?",
-        choices: [
-          { name: "TypeScript", value: "typescript", short: "TypeScript" },
-          { name: "JavaScript", value: "javascript", short: "TypeScript" }, // Both options should have 'TypeScript' as the short value to improve UX and reduce confusion
-        ],
-        default: "typescript",
-      });
-
-      if (language === "javascript") {
-        logger.error("Wrong answer, using TypeScript instead...");
-      } else {
-        logger.success("Good choice! Using TypeScript!");
-      }
-
-      const { packages } = await inquirer.prompt<Pick<CliResults, "packages">>({
-        name: "packages",
-        type: "checkbox",
-        message: "Which packages would you like to enable?",
-        choices: availablePackages
-          .filter((pkg) => pkg !== "envVariables") // dont prompt for env-vars
-          .map((pkgName) => ({
-            name: pkgName,
-            checked: false,
-            // FIXME: TEMPORARY WARNING WHEN USING NODE 18. SEE ISSUE #59
-            disabled:
-              pkgName === "nextAuth" && process.versions.node.startsWith("18")
-                ? "Node.js version 18 is currently not compatible with Next-Auth."
-                : false,
-          })),
-      });
-
-      cliResults.packages = packages;
-
-      // Skip if noGit flag provided
-      if (!cliResults.flags.noGit) {
-        const { git } = await inquirer.prompt<{ git: boolean }>({
-          name: "git",
-          type: "confirm",
-          message: "Initialize a new git repository?",
-          default: true,
-        });
-        if (git) {
-          logger.success("Nice one! Initializing repository!");
-        } else {
-          cliResults.flags.noGit = true;
-          logger.info("Sounds good! You can come back and run git init later.");
+        cliResults.appName = await promptAppName();
```

Comment:
Missing ending bracket here I think? On mobile though so may be wrong?

Reply:
It makes more sense if you try seeing only the last commit

Reply:
I'm also pretty sure that the logic is incorrect on this. Try this:
```
cd cli
pnpm build
pnpm start someAppName
```

Reply:
You are running every prompt conditionally depending on if the name was provided as option

Reply:
@c-ehrlich You are right. I did this while doing my own project, so I lost some context. Fixed now